package.json

```diff
@@ -89,6 +89,9 @@
     "request-promise-native": "1.0.5"
   },
   "devDependencies": {
+    "@babel/core": "7.2.0",
+    "@babel/polyfill": "7.0.0",
```

Comment:
Why not ts-polyfill? It's kept in sync with lib typings.

Reply:
`ts-polyfill` does not work well with IE11 - gives: `Typeerror object doesn't support property or method X`  
I will stick with core-js

Reply:
In the end, ts-polyfill imports actual code from core-js so I don't see how it couldn't work. I managed to run it on BrowserStack: https://imgur.com/cii4Hy8

Reply:
https://imgur.com/a/l0JYahB

Reply:
`equals` is defined in our code, if it fails there this is bug of turnilo, not polypill